node/src/service/service_parachain.rs

```diff
@@ -212,9 +218,11 @@ where
         client.clone(),
     );
 
+    let block_import =
+        ParachainBlockImport::<RuntimeApi, Executor>::new(client.clone(), backend.clone());
```

Comment:
[This](https://github.com/moonbeam-foundation/moonbeam/blob/7f85ea4d5f6feb9e1f37355c8269ffebc533dd51/node/service/src/lib.rs#L466-L467) is what moonbeam uses.

Reply:
Nimbus accepts a [generic implementation of the import queue](https://github.com/moonbeam-foundation/nimbus/blob/1168c153f3113c7f32750c37c367675abaad62ec/nimbus-consensus/src/import_queue.rs#L206). We cannot use [Frontier](https://github.com/moonbeam-foundation/frontier), as Frontier is Moonbeams Ethereum compatibility layer.